dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/ContainerLoaderTest.java

```diff
@@ -0,0 +1,62 @@
+package com.dotcms.rendering.velocity.services;
+
+import com.dotcms.datagen.ContainerAsFileDataGen;
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.repackage.org.apache.commons.io.IOUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.containers.model.FileAssetContainer;
+import com.dotmarketing.util.PageMode;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ContainerLoaderTest {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4945041)